formatter-providers-plugin/src/formatter-providers-frontend-plugin.ts

```diff
@@ -0,0 +1,91 @@
+
+/**
+ * Generated using theia-plugin-generator
+ */
+
+import * as theia from '@theia/plugin';
+
+export function start(context: theia.PluginContext) {
+
+    /**
+     * Register a document formatting provider for all types of files.
+     *
+     * This formatter iterates over each line in the document
+     * and adds the '+' symbol to the beginning of the line.
+     *
+     * Open the Command Palette (press F1 or Ctrl+Shift+P) and run command 'Format Document'
+     */
+    const documentDocumentSelector = {scheme: 'file'};
```

Comment:
Maybe that's enougth `documentSelector`?

Reply:
yes, you're right
There are 3 selectors, so I'll rename them as `documentSector`, `rangeDocumentSelector` and `onTypeDocumentSelector`